aten/src/ATen/core/ivalue.cpp

```diff
@@ -92,9 +92,19 @@ void IValue::getSubValues(HashAliasedIValues& subValues) const {
         pair.key().getSubValues(subValues);
       }
       break;
+    case Tag::Object: {
+      subValues.insert(*this);
+      auto obj_type = type()->expect<ClassType>();
+      auto obj_value = toObject();
+      auto attribute_names = obj_type->attributeNames();
+      for (const auto& name: attribute_names) {
```

Comment:
https://github.com/pytorch/pytorch/pull/32178/files#r385947548 for this comment, could you add a TODO in the comments or address.